open-craft/custom-student-verification
# feat: [BB-4655] Add an admin action to bulk approve users

This PR adds an action to Django Admin, so an admin user can bulk approve multiple Student Verification Requests

Jira ticket: [BB-4655](https://tasks.opencraft.com/browse/BB-4655)

Testing instructions:
- Setup custom-student-verification app with a koa devstack
- Add some verification requests(in my case I created a new test user, and uploaded an image for each one in http://localhost:18000/custom-student-verification)
-  Go to http://localhost:18000/admin/custom_student_verification/studentverificationrequest/ to approve the verification requests
- Select some of this requests, then select the "Mark selected student verification requests as ACCEPTED"
- You should see a "success" message at the top, that contains the amount of requests updated
- Verify that the selecred verification requests have an "ACCEPTED" status
- Verify is signals have been triggered, I think you can do it by checking if the send email signal works, look at `/edx/src/ace_messages` and check that the latest files correspond to "ACCEPTED" messages of the verification requests updated
